src/handlers/proposalsHandlers.ts

```diff
@@ -56,7 +60,160 @@ const getProposals = (
     });
 };
 
-const getProposal = (
+export const getValueWithFullFieldFromRow = (row: ProposalRowWithFieldsAndValues):
+ProposalFieldValue => {
+  if (row.proposalVersionId === undefined
+    || row.proposalVersionId === null
+    || row.proposalVersionApplicationFormId === undefined
+    || row.proposalVersionApplicationFormId === null
+    || row.proposalVersionVersion === undefined
+    || row.proposalVersionVersion === null
+    || row.proposalVersionCreatedAt === undefined
+    || row.proposalVersionCreatedAt === null
+    || row.proposalFieldValueId === undefined
+    || row.proposalFieldValueId === null
```

Comment:
I think about right here is where I would have made a utility method to check all these, lol.

Reply:
Fair enough haha.